pkg/operatorstatus/operatorstatus.go

```diff
@@ -97,8 +97,10 @@ func (o *ClusterOperatorHandler) UpdateOperatorStatus(cfg *v1.Config, deletionIn
 	}
 	if tbrInaccessible {
 		o.setOperatorStatusWithoutInterrogatingConfig(configv1.ConditionFalse, cfg, TBR)
+		metrics.TBRInaccessibleOnBoot(true)
```

Comment:
Is `tbrInaccessible` only set during bootstrap, or is this set every time regardless of the management state?

Reply:
Ah .. answered that in the prior comment .... only during bootstrap